src/main/java/core/basesyntax/Robot.java

```diff
@@ -80,5 +80,14 @@ public void stepForward() {
                 break;
         }
     }
+
+    public boolean isRight(int toX) {
```

Comment:
let's not change Robot implementation, your goal is to solve task with available methods